airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterInput.java

```diff
@@ -37,5 +39,6 @@
   private WorkflowState workflowState;
   private boolean resetConnection;
   private boolean fromJobResetFailure = false;
+  private boolean kaput;
```

Comment:
kaput?

Reply:
I forgot to remove it. It was a field that I added to break the build so I don't forget where changing to a builder was needed. And Kaput = broken in German :D